packages/wiz-ui-react/src/components/custom/chat/components/chat-card.tsx

```diff
@@ -197,7 +196,9 @@ const ChatCard: FC<Props> = ({
                 <WizSelectBox
                   options={statusOptions}
                   value={status}
-                  onChange={(selectedValue) => onChangeStatus?.(selectedValue)}
+                  onChange={(selectedValue) =>
+                    onChangeStatus?.(selectedValue as T)
+                  }
                   placeholder={statusPlaceholder}
                   expand
                 />
```

Comment:
ここの型アサーションは回避できそうでして、例えば、

```suggestion
                <WizSelectBox<T>
                  options={statusOptions}
                  value={status}
                  onChange={(selectedValue) => onChangeStatus?.(selectedValue)}
                  placeholder={statusPlaceholder}
                  expand
                />
```

として、こうするとstatusOptionsの型エラーが出るので、今度はPropsのstatusOptionsを、

```ts
type Props<T> = BaseProps & {
  // 中略
  statusOptions?: ComponentProps<typeof WizSelectBox<T>>["options"];
  // 中略
};
```

とすれば、型アサーションなしで実装できると思います！